packages/odata-service/templates/extend/manifest.json

```diff
@@ -26,7 +26,13 @@
             "<%=model%>": {
                 "dataSource": "<%=name%>",
                 "preload": true,
-                "settings": { }
+                "settings": {<%if (version === '4.0') {%>
```

Comment:
```suggestion
                "settings": {<%if (version === '4') {%>
```

Reply:
Its a string and our versions include a minot version